Automattic/wp-calypso
# GH Integration: Add Card to show post connect UI with deployment information

Closes https://github.com/Automattic/dotcom-forge/issues/1706.

## Proposed Changes

* Add deployment card that gives the user pertinent information about the deployment and allow them to disconnect from branch

## Testing Instructions

- Connect to Github 
- Push a change to your connected branch
- You should see the deployment information displayed on this card (`repo is currently hardcoded`)
-
![image](https://user-images.githubusercontent.com/47489215/220416116-f9d8aa1d-6af5-4c56-89f1-7d6a5e9cbdd9.png)




*

## Pre-merge Checklist
